test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
 			}
+			wg.Wait()
```

Comment:
I can flip it back the other way if that kind of simultaneity is important, but with a 0.2second pause in between each request, almost none of them were ever actually running simultaneously. They mostly seem to take ~2-10 milliseconds to complete when not stepping on each other.